code/obj/machinery/computer/syndicate/podbay.dm

```diff
@@ -0,0 +1,120 @@
+// Riot computer but reflavored for the Cairngorm
+
+/obj/machinery/computer/battlecruiser_podbay
+	name = "podbay authorization"
+	icon_state = "drawbr"
+	desc = "Controls access to the podbay. Use this when your team is ready to go."
+	density = FALSE
+	/// How many authentications are needed to release the shielding on the podbay. Set the first time someone clicks this.
+	var/auth_need
+	var/list/authorized
+
+	light_r = 1
+	light_g = 0.3
+	light_b = 0.3
+
+	/// Whether or not the podbay on the Cairngorm is authorized.
+	var/authed = FALSE
+	/// How long until we take matters into our own hands.
+	var/auth_delay = 10 MINUTES
+
+	New()
+		..()
```

Comment:
```suggestion
```
Unnecessary override; `..()` just calls the parent `New()`, which is implicit behaviour.